core/src/main/scala/org/http4s/rho/bits/ResultResponse.scala

```diff
@@ -1,84 +1,92 @@
 package org.http4s.rho.bits
 
+import cats.data.OptionT
+import cats.{Applicative, Functor, Monad}
```

Comment:
I don't think we need `Applicative`.

Reply:
@bryce-anderson I plan to send a new PR later cleaning up imports all over the place - I'm sure this is not the only place where this leftover happened :)

Reply:
@danielkarch true, will do

Reply:
Any reason not to use  `-Ywarn-unused:imports` and `-Xfatal-warnings`?

Reply:
@danielkarch that is exactly what I'll do in the new PR I mentioned I'll send ;)